Partner Protocols. Doc - 4,5,6,7,9

jengajojo opened the pull request:
Partner Protocols. Doc - 4,5,6,7,9

jengajojo committed:
Create partnerprotocols

jengajojo committed:
Update partnerprotocols

philburrrt commented:
@jengajojo This is a good start. Going to try to expand on it a bit myself. Will update soon.

philburrrt committed:
fix: formatting

file needs to be .md, the single hashtag is the title of the page and double hashtags are headers

lehnberg commented on yearn-ecosystem/collaborative-releases/partnerprotocols.md:
```diff
@@ -0,0 +1,48 @@
+
+Yearn is one of the whitelisted partenrs for the Iron bank, which allows direct levereged lending. Yearn yvaults also serve as a collatral in CREAM along with shared development resources for both sides.
+
+## Cover Protocol
+
+[Cover](https://www.coverprotocol.com/) is decentralized insurance protocol which allows users to hedge against smart contract risks with the help of peer to peer coverage and fungible tokens.
```
Cover is no longer a partner protocol of yearn, suggest we remove this

DarkGhost7 committed:
fix; updates, grammar, delete cover

Deleted cover and fixed grammar / wording

DarkGhost7 committed:
fix: update grammar

DarkGhost7 commented on yearn-ecosystem/collaborative-releases/partnerprotocols.md:
```diff
@@ -0,0 +1,37 @@
+
+**How we collaborate**
+
+Akropolis is the front-of-house institutional service provider for Yearn vaults and lending solutions, offering them bespoke access as well as investment strategies tailored specifically to Yearn products
```
```
front-of-house
```
Sounds odd

DarkGhost7 commented on yearn-ecosystem/collaborative-releases/partnerprotocols.md:
```diff
@@ -0,0 +1,37 @@
+
+## Keep3r Network
+
+[Keep3r Network](https://docs.keep3r.network/) is a coordination protocol for developers to collaborate across different products and strategies. Keep3r facilitates diverse teams to execute common jobs in a decentralized manner. Yearn uses the keep3r network to coordinate with its partners on executing shared strategies across the ecosystem.
```
```is a coordination protocol for developers to collaborate across different products and strategies.```
Is this a good description of keeper?

DarkGhost7 commented on yearn-ecosystem/collaborative-releases/partnerprotocols.md:
```diff
@@ -0,0 +1,37 @@
+
+## Badger
+
+[Badger](https://badger.finance/) is a protocol that facilitates a smooth decentralized bridge for BTC to be brought onto the ETH DeFi ecosystem.
```
```is a protocol that facilitates a smooth decentralized bridge for BTC to be brought onto the ETH DeFi ecosystem.``` 
This isn't badger's main focus just a small extra thing they do. Mainly a yield aggragator.

DarkGhost7 reviewed (changes requested):
Needs to be fixed before it can be added.
